modules/docs/src/main/tut/docs/04-Selecting.md

```diff
@@ -202,7 +212,7 @@ And just for fun, since the `Code` values are constructed from the primary key,
     .query[(Code, Country)]
     .stream.take(5)
     .compile.toList
-    .map(_.toMap)
+    .map(_.toMap) // this causes a deadlock … why?
```

Comment:
Does it still or was this before lambdafy?

Reply:
Good catch, it was that REPL bug. I'll remove that comment.